integration_tests/test_rough_align.py

```diff
@@ -210,6 +211,9 @@ def test_apply_rough_alignment_transform(render, montage_stack, test_do_rough_al
     ex1['minZ'] = 1020
     ex1['maxZ'] = 1022
     ex1['scale'] = 0.1
+    ex1['pool_size'] = 3
```

Comment:
Should we set a test pool size (via environment variable) in `test_data` like we did for render-python?

Reply:
sure.. done